hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieInstant.java

```diff
@@ -67,7 +67,7 @@ public enum State {
     // Committed instant
     COMPLETED,
     // Invalid instant
-    INVALID
+    NIL
```

Comment:
just to confirm no such state can be persisted in any way for actual usage, and it is only meant for testing. By this logic, we should avoid putting this as part of production code, esp. as public API. Can we take this chance to move it to test code?

Reply:
as discussed, scratch the part of moving to test code.